### PR TITLE
Add codeowners file

### PR DESCRIPTION
**Pull request by bhelx**

### CODE REVIEW

**Review by nilslice — APPROVED** [review]
Would also be in support of @chrisdickinson taking ownership, assuming he agrees!

**Comment by chrisdickinson** [comment]
I'd be happy to be added here!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @bhelx @nilslice
```
Comment (bhelx) [inline .github/CODEOWNERS:1]:
```suggestion
* @bhelx @nilslice @chrisdickinson
```

**Comment by bhelx** [comment]
thanks!

**Pull request merged**